idaes/power_generation/properties/__init__.py

```diff
@@ -2,3 +2,7 @@
     FlueGasParameterBlock,
     FlueGasStateBlock,
 )
+from idaes.power_generation.properties.natural_gas_PR import (
+    get_NG_properties,
```

Comment:
Do we really want to add these to the `__init__.py` imports? They are not fully models in their own right, so they are different to the other things here.

Reply:
removed